src/apic-extension/azext_apic_extension/aaz/latest/apic/service/_import_from_apim.py

```diff
@@ -0,0 +1,186 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "apic service import-from-apim",
+)
+class ImportFromApim(AAZCommand):
+    """Imports from APIM instance.
+    """
```

Comment:
Cmd example is better provided here for user to refer to.

Reply:
Added